eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/group/ClientSessionGroupMapping.java

```diff
@@ -56,13 +55,11 @@ public class ClientSessionGroupMapping {
 
     private static final Logger SESSION_LOGGER = LoggerFactory.getLogger("sessionLogger");
 
-    private final ConcurrentHashMap<InetSocketAddress, Session> sessionTable = new ConcurrentHashMap<>();
+    private final ConcurrentHashMap<String/*ip:port*/, Session> sessionTable = new ConcurrentHashMap<>(64);
 
```

Comment:
May I ask that what's the main benefit of changing `InetSocketAddress` to `String`?